utils/admin_console_plugin_index.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+function extractTextsFromPlugin(plugin) {
+    const texts = [];
+    if (plugin.name) {
+        texts.push(plugin.name);
+    }
+    if (plugin.id) {
+        texts.push(plugin.id);
+    }
+    if (plugin.settings_schema) {
+        if (plugin.settings_schema.footer) {
+            texts.push(plugin.settings_schema.footer);
+        }
+        if (plugin.settings_schema.header) {
+            texts.push(plugin.settings_schema.header);
```

Comment:
For `footer` and `header` applies same comment below regarding `help_text` markdown.

Reply:
done 👍

Reply:
I still don't see the change of the markdown for `footer` or `header`. Maybe you missed to push these changes?